docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,175 @@
+
+1. It publishes all operations in the provided manifest file to the PQL linked to the specified variant.
+    - Publishing a manifest to a PQL is additive. Any _existing_ entries in the PQL remain.
+    - If you publish an operation with the same `id` but different details from an existing entry in the PQL, the entire publish command fails with an error.
+
+2. It updates any _other_ variants that the PQL is applied to so that routers associated with those variants can fetch their updated PQL.
+
+As with manifest generation, you can execute this command in your CI/CD pipeline to publish new operations as part of your app release process.
+
+### 2. Configure generated operation models
+
+Both persisted queries and APQs require your code generation to include operation IDs. You can configure this in your code generation configuration's [`options`](output-options).
+Specifically, set the `operationDocumentFormat` array to `definition`, `operationId`, or both `definition` and `operationId`.
+- To use APQs, you must include both the `definition` and `operationId`.
+- For persisted queries, you only need the `operationId`.
+
+```json title="apollo-codegen-config.json"
+"options": {
+    "operationDocumentFormat" : [
+        "definition",
+        "operationId"
+    ]
+}
+```
+
+### 3. Enable persisted queries on `ApolloClient`
+
+Once you've configured your code generation to include operation IDs, you can update your client to query by operation ID rather than the full operation string. This configuration is the same whether you're using APQ or persisted queries:
+
+- Initialize a custom `NetworkTransport` using `RequestChainNetworkTransport` with `autoPersistQueries` parameter set to `true` and an `interceptorProvider` that includes the `AutomaticPersistedQueryInterceptor` (such as `DefaultInterceptorProvider`).
+- Initialize your `ApolloClient` with the custom `NetworkTransport` that supports persisted queries.
+
+```swift
+let store = ApolloStore(cache: InMemoryNormalizedCache())
+let interceptorProvider = DefaultInterceptorProvider(store: store)
+let networkTransport = RequestChainNetworkTransport(
+    interceptorProvider: interceptorProvider,
+    endpointURL: URL(string: "http://localhost:4000/graphql")!,
+    autoPersistQueries: true
+)
+
+let client = ApolloClient(networkTransport: networkTransport, store: store)
+```
+
+For more information on configuring your `ApolloClient`, `NetworkTransport`, `InterceptorProvider`, and the request chain, see the [request pipeline](./../networking/request-pipeline) documentation.
+
+#### Sending APQ retries as `GET` requests
+
+> **Note:** Apollo iOS only retries failed ID-based operations for APQs, not persisted queries.
+
+By default, the Apollo clients sends operation retries as `POST` requests.  In some cases, you may prefer or need to retry an operation using a `GET` request: for example, you may make requests to a CDN that has better performance with `GET`s.
```

Comment:
This isn't about the docs, just the feature — I'm a bit confused as to why iOS can be configured to use GETs for *retries* specifically, vs for all concise requests?

Reply:
I think concise requests are always `GET` by default, full requests are always `POST` by default.

So if you want to change your full requests to `GET` you can. Not sure who does this but we have the same configuration option on Apollo Kotlin. I'd be OK leaving it out of the documentation if we think it's not that useful.